### ISSUE TITLE
Visible action to reset animo demo

### ISSUE THREAD

**Issue by Vickysomtee** [animo/animo-demo#135]
I think we should consider having a reset button for a particular Demo in case a user wants to retake the demo.

My suggestion is either putting a notice of what the `ctrl` + `k`  command does or simply putting a reset button for easy access.

![Screenshot from 2022-11-09 09-06-17](https://user-images.githubusercontent.com/62852943/201088526-9d1579e4-e666-4a39-ba54-2f3ffc8fd2de.png)

**Comment by janrtvld** [comment]
Good suggestion. The 'change' button currently resets the demo. 
<img width="1457" alt="Screenshot 2022-11-10 at 18 24 25" src="https://user-images.githubusercontent.com/60812202/201164721-d980df0a-59e9-487d-b105-bffd1bf56d51.png">

However this is not clear to the user. We could change the label from `change` to `reset`. I would like to keep the `ctrl` + `k` menu hidden as it also has some developer options that might confuse the user.

What do you think?

**Comment by Vickysomtee** [comment]
Okay, changing the `change` to `reset` makes sense but this resets the entire dashboard which in not really a good flow

User experience wise, the `change` button should be used to change the demo character, then we can have a `reset` beside the `start` button of each demo flow to reset that particular flow (If possible).

**Comment by janrtvld** [comment]
Changing a character requires you to receive a new set of starter credentials, so that's why it's currently resetting the whole demo in that flow (there is an issue open to fix this #13). 

If i understand correctly, you mean reset as in resetting the use case so you can re-do it? Currently, there is an option for this in the ctrl + k menu, but maybe we can change it so you can hover over the checkmark of the use case and than re-do it. It will clutter up the user's wallet with more of the same credentials though as we can't delete them from a user's wallet.

**Comment by Vickysomtee** [comment]
Yes, I mean resetting just a use case. Can there be a work around to avoid the same credentials to be clutterd in the user's wallet? If no, we can leave the usecase resetting and then focus on just resetting the character demo

So this is my point:
Can we make resetting the demo possible without hitting the `ctrl + k` because most users are not aware of the `ctrl + k` command and as you stated, you want it hidden. Would be better to have something else apart from the change button (Which might soon serve another purpose if issue #13 is worked on) to reset a character demo.

We can have the reset button opposite the text with a red font. Sounds cool?

<img width="1457" alt="201164721-d980df0a-59e9-487d-b105-bffd1bf56d51" src="https://user-images.githubusercontent.com/62852943/203071595-0c2e99cf-c0cd-408c-9c80-15ad5dab66ec.png">

**Comment by janrtvld** [comment]
We can't really do anything about the cluttered credentials in the user's wallet, but its not that big of a problem.

If you complete the use case it currently shows like this:

<img width="925" alt="image" src="https://user-images.githubusercontent.com/60812202/203577428-87e502a1-4060-4ecd-998b-348b5fc8bf75.png">

Maybe we can keep it like this and just add a rewind icon in there and allow users to redo the use case. There is nothing actually prohibiting users from redoing a use case, besides some information in the state about what use cases are completed.
